spec/api-browser-window-spec.ts

```diff
@@ -5275,6 +5275,22 @@ describe('BrowserWindow module', () => {
         w.closeFilePreview();
       }).to.not.throw();
     });
+
+    it('should not call BrowserWindow show event', async () => {
+      const w = new BrowserWindow({ show: false });
+      const shown = once(w, 'show');
+      w.show();
+      await shown;
+
+      let showCalled = false;
+      w.on('show', () => {
+        showCalled = true;
+      });
+
+      w.previewFile(__filename);
+      await setTimeout(500);
```

Comment:
I couldn't figure out a great way to test this without adding another internal event just for testing. I also tried `await waitUntil(() => !w.isFocused());` but that didn't wait long enough for the second `show`.

I would have also liked to create a test with another window covering the QuickLook window but that doesn't work either.